recipe/build.sh

```diff
@@ -16,7 +16,7 @@ export SAGE_SPKG_INST="$SAGE_LOCAL/var/lib/sage/installed"
 export SAGE_DOC="$SAGE_SHARE/doc/sage"
 
 ln -s "$PREFIX" local
-export SAGE_NUM_THREADS=2
+export SAGE_NUM_THREADS=$CPU_COUNT
```

Comment:
Yeah, this is the right way to do this anyways. We override this value in conda-forge to be equal to the number of CPUs the CI in question provides.